support/doc/plugins/guide.md

```diff
@@ -324,6 +324,7 @@ async function register ({
 
       // You can also return a promise
       return {
+        inputOptions: [],
```

Comment:
Then please add doc for the `videoFilters` param :)

Reply:
During my testing with live transcoding profile I noticed that profiles that specify `videoFilters`  do not generate working `ffmpeg` commands because of the complex filter used for scaling all resolutions.

Should I put a warning in the documentation stating that `videoFilters` must not be used for live transcode filters or should I change the `applyEncoderOptions` to ignore `videoFilters` if it is building a live transcode command ?